jujugui/static/gui/src/app/views/utils.js

```diff
@@ -1513,15 +1513,15 @@ YUI.add('juju-view-utils', function(Y) {
         } else {
           charm = charm[0];
           var config = {},
-              options = charm.get('options');
+              options = charm.get ? charm.get('options') : charm.options;
```

Comment:
I wasn't really sure if this can ever be a Y.Model anymore with the new jujulib changes so I just made it accept both just to be safe.